homeassistant/components/connectedcars/translations/da.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
```

Comment:
Don't include additional translations. Additional language can be maintained via Lokalise after this PR has been merged. Please remove this file.

Reply:
I see, ok I will remove the file.